roles/config-rh-sso/tasks/configure-rh-sso.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- name: "Requesting Access Token"
+  uri:
+    url: "http://10.9.51.153:8080/auth/realms/master/protocol/openid-connect/token"
+    method: POST
+    body_format: form-urlencoded
+    body:
+      username: "{{ default_sso_user }}"
+      password: "{{ default_sso_pass }}"
+      grant_type: "password"
+      client_id: "admin-cli"
+  register: rh_sso_token
+
+- name: "Create Realm for service Red Hat Single Sign-On"
+  uri:
+    url: "http://10.9.51.153:8080/auth/admin/realms"
```

Comment:
Is this the only way to create a realm through the API? Would prefer not to store/manage the `realm.json` file. If the answer is "yes", we should at least look to strip down the .json file to a minimum, preferably just a delta + make it a jinja2 template so values can be substituted.

Reply:
Apparently, yes. According to the information from Red Hat SSO REST API, you can only import a Realm that is represented by a .json file.
The realm in this example was intentionally huge for testing purposes, but I recreated it using the smallest subset of parameters, which is already updated.
I will look into making it a jinja2 template.